pandas/tests/frame/methods/test_to_numpy.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
```

Comment:
shouldn't these be with test_values.py

Reply:
these tests are for to_numpy.  yes the methods are related, but this is the best way to keep the "where are tests for X" unambiguous

Reply:
ok